boards/arm/arduino_portenta_h7/board.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022 Benjamin Björnsson <benjamin.bjornsson@gmail.com>.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <init.h>
+#include <drivers/gpio.h>
```

Comment:
```suggestion
#include <zephyr/init.h>
#include <zephyr/drivers/gpio.h>
```
Cf https://github.com/zephyrproject-rtos/zephyr/pull/45417